### ISSUE TITLE
fix patches link

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2320]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.24.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.24.1...4.24.2) (2021-11-24)

- Release 4.24.2 [`12960ce`](https://github.com/open-wa/wa-automate-nodejs/commit/12960ce41db58c7e2f4fcf7b3bdb0e92f936ed78)
- fix package json patches link #2320 [`81ffbeb`](https://github.com/open-wa/wa-automate-nodejs/commit/81ffbebc0fc85d1d34709594b11ce5090097112a)
- 🥚  patches update [`0e48bf0`](https://github.com/open-wa/wa-automate-nodejs/commit/0e48bf026a4b1ed18e753897b69fb260d1d95138)
- updated types-only package [`b39b62c`](https://github.com/open-wa/wa-automate-nodejs/commit/b39b62c7c40edcb3c65e6a9f5579b2f4742b101a)
